AutomatticTracks/src/main/AndroidManifest.xml

```diff
@@ -9,5 +9,7 @@
         <meta-data
             android:name="io.sentry.auto-init"
             android:value="false" />
+
+        <meta-data android:name="io.sentry.breadcrumbs.user-interaction" android:value="false" />
```

Comment:
Disables a new (enabled by default) feature of Sentry to log every UI interaction like this:
<img width="1011" alt="Screenshot 2022-08-02 at 11 18 46" src="https://user-images.githubusercontent.com/5845095/182348061-d7df0911-936e-44e7-a52f-33cd70b04b73.png">

Reply:
I wonder if it could be useful to use this UI interaction breadcrumbs, but I imagine it can be too overwhelming depending on the scenario. Was that the reason to disable it?

Reply:
I decided to disable it (as almost any other breadcrumb: HTTP, fragment lifecycle) because of privacy concerns (internal link: p91TBi-5Bu-p2)